eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -214,7 +215,10 @@ object NodeParams {
     val overrideFeatures: Map[PublicKey, Features] = config.getConfigList("override-features").asScala.map { e =>
       val p = PublicKey(ByteVector.fromValidHex(e.getString("nodeid")))
       val f = Features.fromConfiguration(e)
-      p -> f
+      val err = Features.validateFeatureGraph(f)
+      require(err.isEmpty, err.map(_.message))
+      require(f.hasFeature(Features.VariableLengthOnion), s"${Features.VariableLengthOnion.rfcName} must be enabled")
```

Comment:
That should probably be factored in a `def validateFeatures`, defined inside `def makeNodeParams`.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1552/commits/778b857a752a6cedf6953b4052e7f55581b47570